Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1412,7 +1408,6 @@ WI.tabContentViewClassForRepresentedObject = function(representedObject)
         return WI.SourcesTabContentView;
 
     if (representedObject instanceof WI.DOMStorageObject || representedObject instanceof WI.CookieStorageObject ||
-        representedObject instanceof WI.DatabaseTableObject || representedObject instanceof WI.DatabaseObject ||
```

Comment:
there's a few other values on `WI.ContentViewCookieType` that probably should be removed